main.go

```diff
@@ -130,6 +131,15 @@ func main() {
 		setupLog.Error(err, "could not create spoke netop client for slice gateway reconciler")
 		os.Exit(1)
 	}
+
+	clientForHubMgr, err := client.New(ctrl.GetConfigOrDie(), client.Options{
+		Scheme: scheme,
+	})
+	//check if user has provided NODE_IP as env variable, if not fetch the ExternalIP from gateway nodes
+	nodeIP, err := cluster.GetNodeIP(clientForHubMgr)
```

Comment:
where does it check `NODE_IP` env?

Reply:
https://github.com/kubeslice/worker-operator/blob/master/pkg/cluster/node.go#L51